lib/plenario_aot/aot_data.ex

```diff
@@ -0,0 +1,41 @@
+defmodule PlenarioAot.AotData do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias PlenarioAot.AotData
+
+  schema "aot_data" do
+    field :aot_meta_id, :id
+    field :node_id, :string
+    field :human_address, :string
+    field :latitude, :float
+    field :longitude, :float
+    field :timestamp, :naive_datetime
+    field :observations, :map
+    field :location, Geo.Point
+    timestamps()
+  end
+
+  def changeset(params) do
```

Comment:
Since admins are the only ones creating and updating these, it's just easier to have a simple changeset here.